Add specs

Zmoki opened the pull request.

Zmoki commented:
@dmitry-mukhin @andrewshkovskii the `specs` folder for `yaml` files with OpenAPI specifications. 

The root contains the script to build HTML static site from specs to `dist` folder. Next, I'll add Travis config to build with that script to the `gh-pages` branch on commit to master.

dmitry-mukhin commented on README.md:
```diff
@@ -1 +1,5 @@
\ No newline at end of file
+# API References of Uploadcare
+
+This repository contains [OpenAPI v3 specifications][openapi] for the Uploadcare API.
```
надо бы написать, что выполнить чтобы собрать всё

Zmoki commented:
@dmitry-mukhin I added checking of yaml formatting, it will be run on each commit with [travis CI](https://travis-ci.org/uploadcare/api-references)

Huge update of README, add info about writing, testing and building to HTML views.

We will not publish this HTML to github page here.

I added this repo as submodule to us frontend and publish it to uploadcare.com/docs/api/, ok?
